<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidgetLeft">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="widgetPlants">
        <attribute name="title">
         <string>Company</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeWidget" name="treeWidgetDB">
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widgetElements">
        <attribute name="title">
         <string>Models</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_addModel">
           <property name="text">
            <string>Add Model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeWidget" name="treeWidgetModel">
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
           <widget class="QTreeWidget" name="treeWidgetElement">
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_addElement">
           <property name="text">
            <string>Add Element</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widgetRoute">
        <attribute name="title">
         <string>Routes</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTreeWidget" name="treeWidgetRoute">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Device</string>
        </attribute>
        <widget class="QPushButton" name="pushButtonRefreshDevice">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>260</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>V-MER List</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidgetDevice">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>191</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
        <widget class="QTabWidget" name="tabWidgetDevice">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>10</y>
           <width>401</width>
           <height>291</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_route">
          <attribute name="title">
           <string>Route</string>
          </attribute>
          <widget class="QPushButton" name="pushButtonRemoveRoutes">
           <property name="geometry">
            <rect>
             <x>54</x>
             <y>230</y>
             <width>91</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Remove Route</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>10</y>
             <width>81</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>All Routes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>10</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Routes in V-MER</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QTreeWidget" name="treeWidgetRoute_2">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>30</y>
             <width>191</width>
             <height>191</height>
            </rect>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
          <widget class="QPushButton" name="pushButtonUploadRoutes">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>230</y>
             <width>81</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Upload Route</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidgetDeviceRoute">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>171</width>
             <height>191</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_data">
          <attribute name="title">
           <string>Measurement Data</string>
          </attribute>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>10</y>
             <width>51</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Data List</string>
           </property>
           <property name="textFormat">
            <enum>Qt::MarkdownText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonDownloadData">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>230</y>
             <width>91</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Download Data</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonRefreshData">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>230</y>
             <width>81</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidgetData">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>30</y>
             <width>191</width>
             <height>191</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidgetRight">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="informations">
        <attribute name="title">
         <string>Informations</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="vLayoutInfoTab"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="properties">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="vLayoutPropTab"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTEST">
    <property name="title">
     <string>TEST</string>
    </property>
    <addaction name="actionECHO_DATABASE"/>
    <addaction name="actionQUERY_ALL_TABLES"/>
    <addaction name="actionReport"/>
    <addaction name="actionGraph"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTEST"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionECHO_DATABASE">
   <property name="text">
    <string>ECHO DATABASE</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionQUERY_ALL_TABLES">
   <property name="text">
    <string>QUERY_ALL_TABLES</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionReport">
   <property name="text">
    <string>Report</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
